frontend/src/app/components/accounts/accounts-table/accounts-table.js

```diff
@@ -39,10 +39,22 @@ const columns = deepFreeze([
 ]);
 
 function _mapAccountToRow(account, currentUser, baseRoute, selectedForDelete) {
-    const { name, isOwner, defaultResource } = account;
+    const { name, defaultResource, deleteRestriction } = account;
     const isCurrentUser = name === currentUser;
     const accountNameText = `${name} ${isCurrentUser ? '(Current user)' : ''}`;
     const usingInternalStorage = defaultResource === 'INTERNAL_STORAGE';
+    let deleteButtonToolTip;
+    switch (deleteRestriction) {
+        case 'OWNER':
+            deleteButtonToolTip = 'Cannot delete system owner';
+            break;
+        case 'OWN_BUCKET':
+            deleteButtonToolTip = 'Cannot delete buckets owner';
+            break;
+        default:
+            deleteButtonToolTip = 'Delete account';
+            break;
+    }
```

Comment:
I think it will be more readable if this is seperate into a function (e.g. `_getDeleteBucketTooltip(deleteRestriction`))